wire-ios/Wire-iOS/Resources/de.lproj/Localizable.strings

```diff
@@ -86,7 +86,6 @@
 "peoplepicker.no_matching_results.action.learn_more" = "Mehr erfahren";
 "peoplepicker.no_matching_results.action.manage_services" = "Dienste verwalten";
 
-"peoplepicker.no_matching_results_after_address_book_upload_title" = "Kein Ergebnis";
```

Comment:
To me it looks like these translations will be lost.

Reply:
address book has been removed